overlord/configstate/configstate_test.go

```diff
@@ -273,15 +273,24 @@ func (wm *witnessManager) Ensure() error {
 }
 
 func (s *configcoreHijackSuite) TestHijack(c *C) {
+	s.state.Lock()
+	defer s.state.Unlock()
+
+	ts := configstate.Configure(s.state, "core", map[string]interface{}{
+		"witness": true,
+	}, 0)
+
 	configcoreRan := false
 	witnessCfg := false
-	witnessConfigcoreRun := func(dev sysconfig.Device, conf configcore.Conf) error {
+	witnessConfigcoreRun := func(dev sysconfig.Device, conf configcore.RunTransaction) error {
 		// called with no state lock!
 		conf.State().Lock()
 		defer conf.State().Unlock()
 		err := conf.Get("core", "witness", &witnessCfg)
 		c.Assert(err, IsNil)
 		configcoreRan = true
+		c.Assert(len(ts.Tasks()), Equals, 1)
```

Comment:
this assert could just go after ts is defined, no? we could also capture a configTask there for use in the next Assert?

Reply:
Good point, I've changed this too